README.md

```diff
@@ -1,3 +1,39 @@
-# Bookchain
+#### Purpose
+This is Bookchain, it is a decentralized virtual library to keep track of your physical library. Library? Yes, we do mean books.
+This app consists of two components, this is the back-end portion. The front-end
+is written in React, and is located here [Bookchain-react](https://github.com/Sh1pley/bookchain-react). This repository is written in Solidity and is built upon the block chain.
 
-### A decentralized virtual library to keep track of your centralized physical library
\ No newline at end of file
+#### Getting Started
+1. This project is built in Solidity, want more info? See the resources section
+   below.
+2. git clone or fork this repository `$ git clone https://github.com/njgheorghita/bookchain.git`
+3. Install Solidity `$ npm install -g solc`
```

Comment:
@njgheorghita I wasn't sure about that. To be honest I couldn't remember, when I added that I was referencing the Solidity docs.